### PR TITLE
string updates

### PR DESCRIPTION
**Pull request by Randy424**
Signed-off-by: randybrunopiverger <rbrunopi@redhat.com>
regarding: https://github.com/stolostron/backlog/issues/25128

### CODE REVIEW

File: frontend/src/lib/doc-util.tsx [hunk 1]
```diff
@@ -13,24 +13,24 @@ export const OCP_DOC_BASE_PATH = 'https://docs.openshift.com/container-platform'
...
+    CREATE_CONNECTION_GCP: `${DOC_BASE_PATH}/multicluster_engine/credentials/credentials#creating-a-credential-for-google-cloud-platform`,
+    CREATE_CONNECTION_VMWARE: `${DOC_BASE_PATH}/multicluster_engine/credentials/credentials#creating-a-credential-for-vmware-vsphere`,
+    CREATE_CONNECTION_OPENSTACK: `${DOC_BASE_PATH}/multicluster_engine/credentials/credentials#creating-a-credential-for-openstack`,
+    CREATE_CONNECTION_VIRTUALIZATION: `${DOC_BASE_PATH}/multicluster_engine/credentials/credentials#creating-a-credential-for-virtualization`,
+    CREATE_CONNECTION_BAREMETAL: `${DOC_BASE_PATH}/multicluster_engine/credentials/credentials#creating-a-credential-for-bare-metal`,
+    CREATE_CONNECTION_ANSIBLE: `${DOC_BASE_PATH}/multicluster_engine/credentials/credentials#creating-a-credential-for-ansible`,
+    CREATE_CONNECTION_REDHATCLOUD: `${DOC_BASE_PATH}/multicluster_engine/credentials/credentials#creating-a-credential-for-openshift-cluster-manager`,
+    CREATE_CONNECTION_PROXY: `${DOC_BASE_PATH}/multicluster_engine/credentials/credentials#proxy`,
```
Comment (swopebe) [inline frontend/src/lib/doc-util.tsx:26]: I am not sure about proxy here. @cadawson will take this over today (Friday)

File: frontend/src/lib/doc-util.tsx [hunk 2]
```diff
@@ -13,24 +13,24 @@ export const OCP_DOC_BASE_PATH = 'https://docs.openshift.com/container-platform'
...
+    CREATE_CONNECTION_PROXY_VMWARE: `${DOC_BASE_PATH}/multicluster_engine/credentials/credentials#proxy-vm`,
+    CREATE_CONNECTION_PROXY_AWS: `${DOC_BASE_PATH}/multicluster_engine/credentials/credentials#proxy-aws`,
+    CREATE_CONNECTION_PROXY_AZURE: `${DOC_BASE_PATH}/multicluster_engine/credentials/credentials#proxy-azure`,
+    CREATE_CONNECTION_PROXY_BAREMETAL: `${DOC_BASE_PATH}/multicluster_engine/credentials/credentials#proxy-bare`,
+    CREATE_CONNECTION_PROXY_GCP: `${DOC_BASE_PATH}/multicluster_engine/credentials/credentials#proxy-google`,
+    CREATE_CONNECTION_PROXY_OPENSTACK: `${DOC_BASE_PATH}/multicluster_engine/credentials/credentials#proxy-openstack`,
+    CREATE_CONNECTION_PROXY_VIRTUALIZATION: `${DOC_BASE_PATH}/multicluster_engine/credentials/credentials#proxy-virtualization`,
     CREATE_CLUSTER: `${DOC_BASE_PATH}/clusters/managing-your-clusters#creating-a-cluster`,
```
Comment (cadawson) [inline frontend/src/lib/doc-util.tsx:34]: `${DOC_BASE_PATH}/multicluster_engine/clusters/managing-your-clusters#creating-a-cluster` ??

File: frontend/src/lib/doc-util.tsx [hunk 3]
```diff
@@ -13,24 +13,24 @@ export const OCP_DOC_BASE_PATH = 'https://docs.openshift.com/container-platform'
...
     CREATE_CLUSTER_ON_PREMISE: `${DOC_BASE_PATH}/clusters/managing-your-clusters#creating-a-cluster-on-premises`,
```
Comment (cadawson) [inline frontend/src/lib/doc-util.tsx:35]: `${DOC_BASE_PATH}/multicluster_engine/clusters/managing-your-clusters#creating-a-cluster-on-premises` ??

File: frontend/src/lib/doc-util.tsx [hunk 4]
```diff
@@ -13,24 +13,24 @@ export const OCP_DOC_BASE_PATH = 'https://docs.openshift.com/container-platform'
...
     IMPORT_CLUSTER: `${DOC_BASE_PATH}/clusters/managing-your-clusters#importing-a-target-managed-cluster-to-the-hub-cluster`,
```
Comment (cadawson) [inline frontend/src/lib/doc-util.tsx:36]: `${DOC_BASE_PATH}/multicluster_engine/clusters/managing-your-clusters#importing-a-target-managed-cluster-to-the-hub-cluster`

File: frontend/src/lib/doc-util.tsx [hunk 5]
```diff
@@ -39,9 +39,9 @@ export const DOC_LINKS = {
     CLUSTER_POOLS: `${DOC_BASE_PATH}/clusters/managing-your-clusters#managing-cluster-pools`,
```
Comment (cadawson) [inline frontend/src/lib/doc-util.tsx:39]: `${DOC_BASE_PATH}/multicluster_engine/clusters/managing-your-clusters#managing-cluster-pools`

File: frontend/src/lib/doc-util.tsx [hunk 6]
```diff
@@ -39,9 +39,9 @@ export const DOC_LINKS = {
...
     CLUSTER_CLAIMS: `${DOC_BASE_PATH}/clusters/managing-your-clusters#claiming-clusters-from-cluster-pools`,
```
Comment (cadawson) [inline frontend/src/lib/doc-util.tsx:40]: `${DOC_BASE_PATH}/multicluster_engine/clusters/managing-your-clusters#claiming-clusters-from-cluster-pools`

File: frontend/src/lib/doc-util.tsx [hunk 7]
```diff
@@ -39,9 +39,9 @@ export const DOC_LINKS = {
...
     SUBMARINER: `${DOC_BASE_PATH}/add-ons/index#submariner`,
-    CONFIG_DISCONNECTED_INSTALL: `${DOC_BASE_PATH}/credentials/credentials#disconnected`,
-    CONFIG_DISCONNECTED_INSTALL_VMWARE: `${DOC_BASE_PATH}/credentials/credentials#disconnected-vm`,
-    CONFIG_DISCONNECTED_INSTALL_OPENSTACK: `${DOC_BASE_PATH}/credentials/credentials#disconnected-openstack`,
+    CONFIG_DISCONNECTED_INSTALL: `${DOC_BASE_PATH}/multicluster_engine/credentials/credentials#disconnected`,
+    CONFIG_DISCONNECTED_INSTALL_VMWARE: `${DOC_BASE_PATH}/multicluster_engine/credentials/credentials#disconnected-vm`,
+    CONFIG_DISCONNECTED_INSTALL_OPENSTACK: `${DOC_BASE_PATH}/multicluster_engine/credentials/credentials#disconnected-openstack`,
     INFRASTRUCTURE_EVIRONMENTS: `${DOC_BASE_PATH}/clusters/managing-your-clusters#creating-an-infrastructure-environment`,
```
Comment (cadawson) [inline frontend/src/lib/doc-util.tsx:45]: `${DOC_BASE_PATH}/multicluster_engine/clusters/managing-your-clusters#creating-an-infrastructure-environment`

File: frontend/src/lib/doc-util.tsx [hunk 8]
```diff
@@ -39,9 +39,9 @@ export const DOC_LINKS = {
...
     DISCOVERED_CLUSTERS: `${DOC_BASE_PATH}/clusters/managing-your-clusters#discovery-intro`,
```
Comment (cadawson) [inline frontend/src/lib/doc-util.tsx:46]: `${DOC_BASE_PATH}/multicluster_engine/discovery/discovery-intro#discovery-intro`

**Review by cadawson — COMMENTED** [review]
Saving

File: frontend/src/lib/doc-util.tsx (hunk 1 above)
Comment (cadawson) [reply frontend/src/lib/doc-util.tsx:26]: I think this one was split by platform.

**Review by cadawson — APPROVED** [review]
Looks good, @Randy424 . Thanks for your help with this!

**Pull request merged**